frontend/stryker.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "./node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+    "_comment": "This config was generated using 'stryker init'. Please see the guide for more information: https://stryker-mutator.io/docs/stryker-js/guides/react",
+    "testRunner": "jest",
+    "reporters": [
+        "progress",
+        "clear-text",
+        "html"
+    ],
+    "coverageAnalysis": "off",
+    "jest": {
+        "projectType": "create-react-app"
+    },
+    "tempDirName": "stryker-tmp",
```

Comment:
It looks like this is set to `.stryker-tmp` by default. ([docs](https://stryker-mutator.io/docs/stryker-js/configuration/#tempdirname-string)) Is there any reason this is being explicitly overridden?

If this is being intentionally overridden, the `.gitignore` should be updated to match.

Reply:
Good catch.   I'll edit this out so we use the default, and change the .gitignore to match that.

Reply:
Fixed